src/image-display.cc

```diff
@@ -120,6 +120,23 @@ static ExifImageOp GetExifOp(Magick::Image &img) {  // NOLINT
     return {};
 }
 
+// An extended version of Magick::readImages that requests a
+// decoding/raster with a transparent background.
+static void readImagesWithTransparentBackground(
+    std::vector<Magick::Image> *sequence, const std::string &image_spec) {
+    MagickLib::ImageInfo *image_info = MagickLib::CloneImageInfo(0);
```

Comment:
I'd probably use `nullptr` here for readability: `MagickLib::CloneImageInfo(nullptr);`
Because that makes clear from context that we're creating a new image_info.

With 0 it sounds like there might be some magic meaning of cloning image info commonly referred to as the 0th image info. Maybe there is a image info 1 and 2 that we can also clone ?

Long story short, always keep as much as possible information in the code to aid readability
(And yes, Stroustrup was [advising to prefer 0 to NULL](https://www.stroustrup.com/bs_faq2.html#null), but (a) he is wrong and (b) now there is nullptr and then even Bjarne agrees)

Reply:
(solved)